PowershellTools.Common/Constants.cs

```diff
@@ -95,5 +95,25 @@ internal static class Constants
         /// PowerShell host process window title format
         /// </summary>
         public const string HostProcessWindowTitleFormat = "{0} : {1}";
+

```

Comment:
These items should be inside Resources, as they may be presented to Users, and may eventually need to be localized.